src/utils.js

```diff
@@ -325,10 +334,15 @@ const getPlatformTools = () => {
 
 const getNativePlatformTools = () => {
```

Comment:
humm, this name does not make sense anymore does it? since it does not only getNativePlatfromTools anymore. Also would be an good idea to check if the costomTools exists

Reply:
Well, custom tools are still native. On windows you don't have a path, so you kind of need it, if you want to use native tools. I'd keep the name.